### ISSUE TITLE
Remove Documentation/Targets.rst?

### ISSUE THREAD

**Issue by nito-web** [TYPO3-Documentation/TYPO3CMS-Guide-Installation#33]
I cant see what `Documentation/Targets.rst` is there for - could we remove it?

**Comment by marble** [comment]
① Linktargets gives an overview about what labels have been set in the manuals. Very useful if you want to link to the manual or define new labels.
② We could remove it, but we shouldn't :-)) The way it was written it was broken though.
③ I've added a Sitemap now: https://docs.typo3.org/typo3cms/InstallationGuide/Sitemap/Index.html
④ I fixed the Linktargets page: https://docs.typo3.org/typo3cms/InstallationGuide/Linktargets/Index.html

You find some explanations in the source of that Linktargets page as well:
https://docs.typo3.org/typo3cms/InstallationGuide/_sources/Linktargets/Index.txt

Thanks for reporting the problem.
